src/test/kotlin/Command/base/ProcessorTest.kt

```diff
@@ -0,0 +1,32 @@
+package Command.base
+
+import Command.commands.*
+import junit.framework.Assert.assertTrue
+import org.hamcrest.core.Is.`is`
+import org.junit.Assert.assertThat
+import org.junit.Test
+import java.util.*
+
+
+class ProcessorTest {
+
+  val cart = Cart(Scanner(System.`in`))
+  val commands = listOf(
+    ChooseIngredientCommand(cart),
+    ChooseNoodlesCommand(cart),
+    ChooseSauceCommand(cart),
+    ShowPriceCommand(cart)
+  )
+  val processor = Processor(commands, CommandNotFound())
+
+  @Test
+  fun `check if command matcher matches correctly`() {
```

Comment:
In this case if you are write: check if command matcher matches correctly .

- What command?
- What is matched?

Also when you see the code before read this "title": 

```
assertTrue(processor.process("2") is ChooseIngredientCommand)
```

I don't see any matches 👀 , I only see that if I process a "2" command should return a ChooseIngredientCommand. WOW have you seen that? unintentionally I have written the test name 👍